### ISSUE TITLE
Stock availability update doesn't update

### ISSUE THREAD

**Issue by Doraemon74** [tzyganu/Switcher#147]
I try to explain the problem:
I have e configurable product with two products associated. One is available and the other unavailable. In the front-page look right with one option available and the other unavailable. If I change the quantity and the status in Available of the second product in the front-page still unavailable until I enter in the configurable product and save it (even without change anything). 

So in other words,any change of availability are not updated in the fron-page until I don't save the configurable products that are associated. 

Any idea?

**Comment by tzyganu** [comment]
@Doraemon74 You need to rebuild the indexes each time you change the stock status. Let me know if that solves the problem.

**Comment by Doraemon74** [comment]
Hi Marius,
i did it, but didnt wordk. The only way is to save again the configurable
product :/

Gianluca

**Comment by tzyganu** [comment]
@Doraemon74 I don't think the problem is on the extension side. I just use what Magento already has for showing or hiding out of stock products.

**Comment by Doraemon74** [comment]
I know! I just wanted expose my problem, maybe You could help me.

Anyway, I will try to look for a solution.

Thank you,
Gianluca

**Comment by Doraemon74** [comment]
Sorry to disturb you again,
I tried to disable the plugin and I discover that even if a product is "Out
of stock" is showed as well but if you try to put in the cart give an error
"The product is unavailable"... any idea?

just a simple question.

How do you remove the "configurable_swatch_taglia2" block? By CSS? Or is
possible by plugin?

Gianluca

**Comment by tzyganu** [comment]
What is `configurable_swatch_taglia2` block? That's not from my extension

**Comment by tzyganu** [comment]
Closed due to lack of activity.
